workspace/viewer/PdfJsViewer.ts

```diff
@@ -394,14 +399,13 @@ export class PdfJsViewer {
         this.zoom.onResize();
       }
       this.currentTask = task;
-      this.currentPage = task.page;
       const onDocumentSuccess = (pdf) => {
         this.currentFile = pdf;
         this.currentFileUrl = task.url;
         if (this.currentPage > pdf.numPages) {
           this.currentPage = pdf.numPages;
         }
-        this.openPage(pdf, this.currentPage);
+        this.openPage(pdf, this.currentTask.page);
```

Comment:
```
workspace/viewer/PdfJsViewer.ts(408,28): error TS2532: Object is possibly 'undefined'.
```

`currentTask` is allowed to be undefined and at this moment it can become undefined or assigned some other value. If you meant to use `page` from `task` above, please use `task.page`. If you really meant to use `page` from `currentTask`, whatever it is, you need to check if `currentTask` is still defined.

Reply:
Done.